File: problems/280/solution.py
a = int(input())
b = int(input())
c = int(input())

if(a * a + b * b == c * c):
    print("YES")
elif(a * a + c * c == b * b):
    print("YES")
elif(b * b + c * c == a * a):
    print("YES")
else:
    print("NO")

File: problems/280/solution.cpp
#include<iostream>

int main()
{
    int a, b, c;
    std::cin >> a >> b >> c;
    if(a * a + b * b == c * c)
        std::cout << "YES";
    else if(a * a + c * c == b * b)
        std::cout << "YES";
    else if(b * b + c * c == a * a)
        std::cout << "YES";
    else
        std::cout << "NO";
    
	return 0;
}